titiler/api/endpoints/mosaic.py

```diff
@@ -11,10 +12,37 @@
 from titiler.models.mosaic import CreateMosaicJSON, UpdateMosaicJSON
 
 from fastapi import APIRouter, Query
+from fastapi.routing import APIRoute
 
+from starlette.exceptions import HTTPException
+from starlette.requests import Request
 from starlette.responses import Response
 
-router = APIRouter()
+
+class MosaicJSONRouter(APIRoute):
```

Comment:
Including stac backend complicates things for initial development.  Fully anticipate to remove this in the future.